scripts.js

```diff
@@ -0,0 +1,109 @@
+// scripts.js
+
+$(document).ready( function() {
+
+
+// Trek API url to 'get'?
+var url = 'https://trektravel.herokuapp.com/trips';
+$('#trips').hide();
+
+// Print out a list of response links
+var successCallback = function (response) {
+  $('#trips').show();
+
```

Comment:
I would show the div **after** you load it with HTML.  Otherwise you'll see if flash with content.